tests/trainer/test_checkpoint.py

```diff
@@ -487,7 +517,7 @@ def get_logger(self, tmp_path: pathlib.Path):
             num_concurrent_uploads=1,
             use_procs=False,
             upload_staging_folder=str(tmp_path / 'staging_folder'),
-        )
+         )
```

Comment:
```suggestion
        )
```
typo? why is there a space added here?